app/controllers/apicasso/apidocs_controller.rb

```diff
@@ -475,8 +482,9 @@ class ApidocsController < Apicasso::ApplicationController
       end
     end
 
+    # Method that generates Swagger JSON
```

Comment:
That serves the generated Swagger JSON... This is not the method that generates the JSON